apps/namada-interface/src/App/AccountOverview/DerivedAccounts/DerivedAccounts.tsx

```diff
@@ -151,7 +124,11 @@ const DerivedAccounts = (): JSX.Element => {
                       )}
                     </DerivedAccountType>
                   </DerivedAccountInfo>
-                  <FiatBalanceDisplay balance={balance} />
+                  <div className="flex items-center">
+                    <span className="text-white">
+                      {Tokens[symbol].symbol} {nativeBalance?.toString()} 
```

Comment:
I think you can remove `?.toString()` here since `nativeBalance` is of type `string`.